dispatchrouter/pom.xml

```diff
@@ -94,7 +94,7 @@
               <images>
                 <image>
                   <build>
-                    <from>eclipsehono/qpid-dispatch:${qpid-dispatch.version}</from>
+                    <from>enmasseproject/qdrouterd:0.8.0</from>
```

Comment:
It is an official image and it seems to work properly with IT tests. I'll get more info about it and report back.